marklogic-data-hub-central/ui/src/config/test-data.config.js

```diff
@@ -173,40 +173,40 @@ const entityTypeProperties = [
   { key: 9, name: 'gender', type: 'string' }
 ];
 
-const testJSONResponse = { 
-  properties: { 
-    propName: { output: '123EAC', sourcedFrom: 'proteinId' },
+const testJSONResponse = {
+  properties: {
+    propName: { output: 'user1@marklogic.com,user2@marklogic.com, ... (300 more)', sourcedFrom: 'proteinId' },
```

Comment:
can we create a separate test and use an inline jsonResponse object particular to the test. This is going to affect the sorting tests. Are all the unit tests passing with this change locally?

Reply:
This test can rerender with 3 different responses like the screenshots above.

Reply:
Only 2 pertain to this bug for data type array. First screenshot doesn't apply. I have added tests with data pertaining only to this bug